overrides/main.html

```diff
@@ -33,5 +33,5 @@
   <meta name="twitter:image" content="https://dietpi.com/images/dietpi-logo_720x720.png">
 {% endblock %}
 {% block announce %}
-  <a href="https://dietpi.com/contribute.html">Learn how to support and contribute to the DietPi project <span class="twemoji">{% include ".icons/fontawesome/regular/heart.svg" %}</span></a>
+  <a href="https://dietpi.com">WEBSITE</a>&emsp;<a href="https://dietpi.com/#features">FEATURES</a>&emsp;<a href="https://dietpi.com/#download">DOWNLOAD</a>&emsp;<a href="https://dietpi.com/forum/">FORUM</a>&emsp;<a href="https://dietpi.com/blog/">BLOG</a>&emsp;<a href="https://dietpi.com/#gettingstarted">GETTING STARTED</a>&emsp;<a href="https://dietpi.com/contribute.html">DONATE</a>
```

Comment:
```suggestion
  <a href="https://dietpi.com">HOME</a>&emsp;<a href="https://dietpi.com/#features">FEATURES</a>&emsp;<a href="https://dietpi.com/#download">DOWNLOAD</a>&emsp;<a href="https://dietpi.com/forum/">FORUM</a>&emsp;<a href="https://dietpi.com/blog/">BLOG</a>&emsp;<a href="https://dietpi.com/#gettingstarted">DOCS</a>&emsp;<a href="https://dietpi.com/contribute.html">DONATE</a>
```

Reply:
But then I suggest to not open the "Getting started" section of the main website, but stay in the docs, when clicking DOCS 🙂:
```suggestion
  <a href="https://dietpi.com">HOME</a>&emsp;<a href="https://dietpi.com/#features">FEATURES</a>&emsp;<a href="https://dietpi.com/#download">DOWNLOAD</a>&emsp;<a href="https://dietpi.com/forum/">FORUM</a>&emsp;<a href="https://dietpi.com/blog/">BLOG</a>&emsp;<a href="https://dietpi.com/docs/">DOCS</a>&emsp;<a href="https://dietpi.com/contribute.html">DONATE</a>
```

Reply:
DOCS is redundant, and I suggest changing to GITHUB, similar to the main website.
To avoid having a double HOME, I suggest to keep DIETPI HOME

![image](https://github.com/MichaIng/DietPi-Docs/assets/5551796/1da1e682-e022-4dd4-8e89-61b7af70a5b4)

Reply:
@MichaIng could you help me on css, on having the same hover effect?